submission-scripts/script-templates/magpie-hive

```diff
@@ -0,0 +1,92 @@
+#############################################################################
+# Hive configurations
+#############################################################################
+#
+# Should Hive be run
+#
+# Specify yes or no.  Defaults to no.
+#
+export HIVE_SETUP=yes
+
+# Version
+
+export HIVE_VERSION="HIVEVERSIONDEFAULT"
+
+export HIVE_PORT=HIVEDEFAULTPORT
+
+# Path to your Hive build/binaries
+#
+# This should be accessible on all nodes in your allocation. Typically
+# this is in an NFS mount.
+#
+# Ensure the build matches the Hadoop/HDFS version this will run against.
+#
+export HIVE_HOME="HIVEDIRPREFIX/apache-hive-${HIVE_VERSION}-bin"
+
+# Path to store data local to each cluster node, typically something
+# in /tmp.  This will store local conf files and log files for your
+# job.  If local scratch space is not available, consider using the
+# MAGPIE_NO_LOCAL_DIR_DIR option.  See README for more details.
+
+export HIVE_LOCAL_DIR="LOCALDIRPREFIX/hive"
+
+# Hive Clients will make a connection to a metastore server which in turn queries the datastore (PostgreSQL)
+# for metadata. This sets the location of the metastore
+export HIVE_METASTORE_DIR="LUSTREDIRPREFIX/hive/database/postgres/data"
+
+# Configurations for Postgres, used as a backend for HiveServer/Hive Metastore
+export POSTGRES_HOME="POSTGRESDIRPREFIX/pgsql"
+export PGPORT=POSTGRESDEFAULTPORT
+export PGDATA="LUSTREDIRPREFIX/hive/database/postgres/data"
+
+# DB name for this project
+export HIVE_DB_NAME=HIVEDEFAULTDBNAME
+
+###########################################################################
+# Hive Job/Run Configurations
+############################################################################
+
+# Set hive job for MAGPIE_JOB_TYPE = hive
+# "testbench" - to run benchmark testing for Hive via hive-testbench modules
+#
+# "interactive" - manually interact to submit jobs, peruse Hbase, etc.
+#                 In this mode you'll login to the cluster node that
+#                 is your 'master' node and interact with Hbase
+#                 directly (e.g. bin/hbase ...)
+#
+# "setuponly" - Like 'interactive' but only setup conf files. useful
+#               if user wants to setup & teardown daemons themselves.
+#
+# "checkhiveup" - verifies that Hive and Hive Metastore are configured properly
+#
+
+export HIVE_MODE="testbench"
+
+# The following are configurations for running the hive-testbench software
+#
+# Path to the hive-testbench software
+export HIVE_TESTBENCH_DIR="TESTBENCHDIRPREFIX/hive-testbench-hdp3"
```

Comment:
1 question/nit, why does this path have the "-hdp3" extension to it?  Every build I see only just ends with "hive-testbench".

I notice no download path for hive-testbench in README.hive.  Is there a specific build you use/downloaded?